bin/ofmt.js

```diff
@@ -10,14 +10,11 @@ void (async function () {
     flags: {lint: {type: 'boolean', alias: 'l'}},
   })
 
-  const prettierOptions =
-    '--print-width 120 --tab-width 2 --no-semi --single-quote --quote-props as-needed --trailing-comma all --no-bracket-spacing --arrow-parens always'
-
   if (args.flags.lint) {
-    await execAsync(`npx prettier --check ${prettierOptions} ${args.input}`)
+    await execAsync(`npx prettier --check ${args.input}`)
```

Comment:
@quesabe can you please confirm that you tested that this command will pick up the config file from the local dir and not from the calling project dir?

Reply:
Yes, I have tested, and the answer depends on what you mean in `local dir`.
Here's a note from docs:
`The configuration file will be resolved starting from the location of the file being formatted, and searching up the file tree until a config file is (or isn’t) found.`

Thus a config file needs be somewhere on the search path from a tested file.
BTW, I thought that the introduction of install-command is meant to solve this problem.

Reply:
Ok, got it!

Reply:
@quesabe actually not ok 🤓 The config file needs to be specified in `prettier` call if it was not set in the calling project's `package.json`. Otherwise `ofmt` won't work without `ofmt install` (which should be optional).

Reply:
@gvidon 
Just to clarify the expected behaviour:
- if a calling project uses _ofmt_, is it supposed to always use _ofmt_ config for _prettier_ or any other config is allowed?

Especially I'm worried about this scenario:
- _ofmt_ installed (through _npm_, but `ofmt install` not run);
- the project already has its own _.prettierrc_;
- when _ofmt_ is run does it use _ofmt_ config or the project's config?

If it is supposed to always run with _ofmt_ config, it might be better to hardcode the config from _ofmt_ and forget about transferring the one to a project root. According to docs it can even prevent waisting some cpu cycles on config search:
https://prettier.io/docs/en/cli.html#--find-config-path-and---config

And if we plan to respect configs that projects have and don't want to override it with the _ofmt_ one, then a I'll add a check for existing config the _ofmt_ bin.

Reply:
@quesabe
* `ofmt` always uses **only** its own config. Formatting rules **can not** be extended or modified in any way
* The user might call `ofmt` without prior installation with `ofmt install`.
  Thus the prettier config won't be registered in the calling project's `package.json`, and `prettier` won't be able to take its config from there. For this reason the internal call for prettier must always specify the path to the config. It must literally be hardcoded.

We are adding the prettier config into the calling project `package.json` for the only reason of IDE integration.